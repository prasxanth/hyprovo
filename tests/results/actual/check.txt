2021-11-24 14:41:21.178 - INFO - Pass ✓...  add pass
2021-11-24 14:41:21.181 - ERROR - Fail ✗...  sub fail
2021-11-24 14:41:21.184 - ERROR - Fail ✗...  div fail
