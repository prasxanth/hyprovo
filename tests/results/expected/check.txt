2021-11-24 14:10:30.690 - INFO - Pass ✓...  add pass
2021-11-24 14:10:30.693 - ERROR - Fail ✗...  sub fail
2021-11-24 14:10:30.695 - ERROR - Fail ✗...  div fail
